Homework/Hoang_Nguyen/lesson 1/202_happy_number.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {number} n
+ * @return {boolean}
+ */
+var isHappy = function (n) {
```

Comment:
can improve to O(1) extra space?

Reply:
I updated my solution using fast and slow pointers. Please take a look.